INFO  2016-06-19 23:37:36 at Server:201 :		 Server initialized.
INFO  2016-06-19 23:37:48 at Server:154 :		 Connection received
ERROR 2016-06-19 23:38:49 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:38:49 at Session:143 :		 Connection terminated
INFO  2016-06-19 23:39:10 at Server:154 :		 Connection received
INFO  2016-06-19 23:39:16 at Session:208 :		 DSI logged in successfully.
WARN  2016-06-19 23:43:31 at MessageHandler:1302 :		 SQLException caught
java.sql.SQLSyntaxErrorException: ORA-00936: missing expression

	at oracle.jdbc.driver.SQLStateMapping.newSQLException(SQLStateMapping.java:91)
	at oracle.jdbc.driver.DatabaseError.newSQLException(DatabaseError.java:133)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:206)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:455)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:413)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:1034)
	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:183)
	at oracle.jdbc.driver.T4CStatement.executeForDescribe(T4CStatement.java:780)
	at oracle.jdbc.driver.T4CStatement.executeMaybeDescribe(T4CStatement.java:855)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1186)
	at oracle.jdbc.driver.OracleStatement.executeQuery(OracleStatement.java:1377)
	at oracle.jdbc.driver.OracleStatementWrapper.executeQuery(OracleStatementWrapper.java:387)
	at server.MessageHandler.handleEvolutionOfTheSimulation(MessageHandler.java:1270)
	at server.Session.handleMessage(Session.java:245)
	at server.Session.run(Session.java:118)
WARN  2016-06-19 23:43:32 at MessageHandler:1302 :		 SQLException caught
java.sql.SQLSyntaxErrorException: ORA-00936: missing expression

	at oracle.jdbc.driver.SQLStateMapping.newSQLException(SQLStateMapping.java:91)
	at oracle.jdbc.driver.DatabaseError.newSQLException(DatabaseError.java:133)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:206)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:455)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:413)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:1034)
	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:183)
	at oracle.jdbc.driver.T4CStatement.executeForDescribe(T4CStatement.java:780)
	at oracle.jdbc.driver.T4CStatement.executeMaybeDescribe(T4CStatement.java:855)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1186)
	at oracle.jdbc.driver.OracleStatement.executeQuery(OracleStatement.java:1377)
	at oracle.jdbc.driver.OracleStatementWrapper.executeQuery(OracleStatementWrapper.java:387)
	at server.MessageHandler.handleEvolutionOfTheSimulation(MessageHandler.java:1270)
	at server.Session.handleMessage(Session.java:245)
	at server.Session.run(Session.java:118)
ERROR 2016-06-19 23:43:47 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:43:47 at Session:143 :		 Connection terminated
INFO  2016-06-19 23:46:48 at Server:154 :		 Connection received
INFO  2016-06-19 23:46:57 at Session:208 :		 DSI logged in successfully.
ERROR 2016-06-19 23:48:31 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:48:31 at Session:143 :		 Connection terminated
INFO  2016-06-19 23:49:33 at Server:154 :		 Connection received
INFO  2016-06-19 23:49:37 at Session:208 :		 DSI logged in successfully.
ERROR 2016-06-19 23:50:05 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:50:05 at Session:143 :		 Connection terminated
INFO  2016-06-19 23:50:15 at Server:154 :		 Connection received
ERROR 2016-06-19 23:50:22 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:50:22 at Session:143 :		 Connection terminated
INFO  2016-06-19 23:50:25 at Server:154 :		 Connection received
ERROR 2016-06-19 23:50:31 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:50:31 at Session:143 :		 Connection terminated
INFO  2016-06-19 23:50:34 at Server:154 :		 Connection received
INFO  2016-06-19 23:50:37 at Session:208 :		 DSI logged in successfully.
ERROR 2016-06-19 23:51:02 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:51:02 at Session:143 :		 Connection terminated
INFO  2016-06-19 23:52:31 at Server:154 :		 Connection received
INFO  2016-06-19 23:52:34 at Session:208 :		 DSI logged in successfully.
INFO  2016-06-19 23:54:36 at Server:154 :		 Connection received
INFO  2016-06-19 23:54:39 at Session:208 :		 DSI logged in successfully.
ERROR 2016-06-19 23:55:59 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:55:59 at Session:143 :		 Connection terminated
INFO  2016-06-19 23:56:52 at Server:154 :		 Connection received
INFO  2016-06-19 23:56:57 at Session:208 :		 DSI logged in successfully.
ERROR 2016-06-19 23:57:43 at Session:135 :		 Session : IOException caught
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at java.io.BufferedReader.readLine(Unknown Source)
	at server.Session.run(Session.java:111)
INFO  2016-06-19 23:57:43 at Session:143 :		 Connection terminated
